MetaCell/salk-interactive-atlas
# Feature/90

Closes #90

Implemented solution: 
- Adds has_edit_permission as a serializer method field to experiment serializer
- Uses new field to understand if we show the population color picker or not
- Uses regex to identify if the color hex is a valid color or not. If not uses black as default color

How to test this PR: 
- Add new population with invalid color; Verify that the color saved is the default color
- Create public experiment; Log in as a User that is not owner of the experiment; Verify that the population color is only informative (you can't see the color picker to change color)

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
